plugins/LadspaEffect/caps/CMakeLists.txt

```diff
@@ -7,7 +7,10 @@ IF(LMMS_BUILD_WIN64)
 ADD_DEFINITIONS(-DLMMS_BUILD_WIN64)
 ENDIF(LMMS_BUILD_WIN64)
 SET_TARGET_PROPERTIES(caps PROPERTIES PREFIX "")
-SET_TARGET_PROPERTIES(caps PROPERTIES COMPILE_FLAGS "-O2 -funroll-loops -Wno-write-strings")
+
+IF(NOT MSVC)
+	SET_TARGET_PROPERTIES(caps PROPERTIES COMPILE_FLAGS "-O2 -funroll-loops -Wno-write-strings")
+ENDIF()
```

Comment:
Same here. Optimizations could be used for MSVC.

Reply:
Shouldn't have resolved these.

Reply:
done with the latest commit